src/components/Navbar/navbar.css

```diff
@@ -0,0 +1,62 @@
+header{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 10vh;
+    background-color: black;
+}
+
+ ul li{
+    display: inline-block;
+    margin: 2rem;
+}
+.logo{
+    font-size: xx-large;
+    font-family:Verdana, Geneva, Tahoma, sans-serif;
+    font-weight: 600;
+    color:blueviolet
+}
+.searchbar{
```

Comment:
you can give here text color to white because by default text color is black and you already given background as black color